compiler/aarch64/codegen/ARM64Instruction.hpp

```diff
@@ -711,11 +711,41 @@ class ARM64AdminInstruction : public TR::Instruction
     * @param[in] cg : CodeGenerator
     */
    ARM64AdminInstruction(TR::InstOpCode::Mnemonic op, TR::Node *node, TR::Node *fenceNode,
-                          TR::Instruction *precedingInstruction, TR::CodeGenerator *cg)
+                         TR::Instruction *precedingInstruction, TR::CodeGenerator *cg)
       : TR::Instruction(op, node, precedingInstruction, cg), _fenceNode(fenceNode)
       {
       }
 
+   /*
+    * @brief Constructor
+    * @param[in] op : instruction opcode
+    * @param[in] cond : register dependency conditions
+    * @param[in] node : node
+    * @param[in] fenceNode : fence node
```

Comment:
Can you provide more details on what the fence node is and what a consumer of this instruction is expected to provide?

Reply:
There are examples in omr/compiler/arm/codegen/OMRTreeEvaluator.cpp passing a fence node to `generateAdminInstruction()`.  The fence node comes from `TR::Node::createRelative32BitFenceNode()` .  The p version of OMRTreeEvaluator.cpp does it in a similar way.

Reply:
OK.  I studied this code a bit more and my confusion came from the overloaded nature of an "admin" instruction.  A fence node is only needed for (surprise!) fence instructions, and it's a fabricated node to hold the kind of relocation represented by the fence.  I believe using a node rather than some other structure is historical.  Anyway, I'm fine with this.